ZygorGuidesViewer.lua

```diff
@@ -199,6 +252,48 @@ Z_SKIN_HOOKS = {
 		ZygorGuidesViewerFrame_Border_Logo:SetTexture(nil)
 		ZygorGuidesViewerFrame_Border_Top:ClearAllPoints()
 	end,
+	Classic = function()
+		SKINDIR = DIR.."\\Skin\\Classic" -- Force the path to your Classic folder

```

Comment:
Can you use a local variable instead ? I'm not sure it's a good idea to overwrite SKINDIR here

Reply:
Shouldn't be even needed, `SKINDIR = DIR.."\\Skin\\Classic"` should already equal this. If it isn't yet than something is wrong.
```function me:UpdateSkin()
  SKINDIR = DIR .. '\\Skin\\' .. self.db.profile.skin```
  Is a few lines above what calls Classic func.
  ```  choice = self.skin_choice[self.db.profile.skin]
  if choice then
    LoadSkin(choice)
  else
    --LoadDefault()
    self:Print('Load skin Error!')
  end```